charts/pvc-autoresizer/templates/controller/certificate.yaml

```
@@ -1,5 +1,6 @@
 {{- if not .Values.webhook.caBundle }}
 {{- if not .Values.webhook.existingCertManagerIssuer }}
+{{- if (index .Values "cert-manager" "enabled") }}
```

There are cases pvc-autoresizer uses existing cert-manager without installing cert-manager. Therefore, I don't suggest using `.Values.cert-manager.enabled` as a condition for the certificate and issuer. On the other hand the newly suggested `.Values.webhook.certificate.generate` is appropriate.

```suggestion
{{- if not .Values.webhook.certificate.generate }}
```